consumer_test.go

```diff
@@ -33,6 +33,14 @@ var _ = Describe("Consumer", func() {
 		}()
 	}
 
+	It("should fail for invalid group id", func() {
+		_, err := newConsumer("")
+		Expect(err).To(HaveOccurred())
+
+		_, err = newConsumer("foo:bar")
```

Comment:
I assume your implementation is based on https://github.com/apache/kafka/blob/b62f8ea43b6d5307f7274fbe8b7984dd5ee22239/core/src/main/scala/kafka/common/Config.scala#L26. Unfortunately, we ourselves are using `:` in group names without problems. Could I ask you to keep them in for backwards compatibility?